### PR TITLE
Update endpoint-dlp-getting-started.md

### PR DESCRIPTION
**Pull request by msbemba**
Added - Global reader, Security reader, and  Reports reader as they are able to view activity explorer. (They cannot turn on auditing but can view the date in activity explorer)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3707

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 1]
```diff
@@ -72,6 +72,9 @@ Data from Endpoint DLP can be viewed in [Activity explorer](data-classification-
 - Compliance admin
 - Security admin
 - Compliance data admin
+- Global reader
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md:75]:
```suggestion
- Global reader (view only)
```

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 2]
```diff
@@ -72,6 +72,9 @@ Data from Endpoint DLP can be viewed in [Activity explorer](data-classification-
...
+- Security reader
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md:76]:
```suggestion
- Security reader (view only)
```

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 3]
```diff
@@ -72,6 +72,9 @@ Data from Endpoint DLP can be viewed in [Activity explorer](data-classification-
...
+- Reports reader
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md:77]:
```suggestion
- Reports reader (view only)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions.

**Comment by msbemba** [comment]
> Copy review - suggestions.
Hi Johan, I don't think an explicit mention of View only is needed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**